### ISSUE TITLE
json parsing error in ensurettlindex function

### ISSUE THREAD

**Issue by pritamlipu1** [arangodb/go-driver#393]
I am using V1.3.1

Issue found in versions: v1.3.0 and v1.3.1

When I am calling  -->. coll.EnsureTTLIndex(ctx, "updatedAt", InactiveTimeLimitSec, nil)

where InactiveTimeLimitSec = 30*24*3600       //30 days

Its giving error --> ttl index(updatedAt): json: cannot unmarshal number 0.5714285714285714 into Go value of type int

Please help

**Comment by jwierzbo** [comment]
What version of ArangoDB are you using?
I've just run a simple test with the above value with success on 3.9.0:

```
col, err = db.CreateCollection(ctx, "ttl", nil)
idx, created, err := col.EnsureTTLIndex(nil, "updatedAt", 30243600, nil)
```

**Comment by pritamlipu1** [comment]
@jwierzbo in docker, m fetching latest arango.
eg: 
arangodb:
    image: arangodb:latest

https://github.com/arangodb/go-driver version: 
version before v1.3.0 working.
In v1.3.0 and v1.3.1, I am getting above error.

**Comment by jwierzbo** [comment]
@pritamlipu1 we have problem with reproduction of this issue. 

We have already E2E test here: http://github.com/arangodb/go-driver/blob/477d962f231a78126d60a0be40d7c1d3db4f12a0/test/indexes_test.go#L297-L297

You can change `expireAfter` value to `30243600` and then run that test with these commands:

```
export ARANGODB=arangodb/arangodb:latest
TESTOPTIONS="-test.run TestEnsureTTLIndex -test.v" make run-tests-single-json-no-auth
```

Could you check if this is failing for you?

**Comment by pritamlipu1** [comment]
@jwierzbo Please try to reproduce with following versions
Please use https://github.com/arangodb/go-driver/releases/tag/v1.3.1
To start arango plz use docker. eg:
`  arangodb:
    image: arangodb:latest
    ports:
      - 8529:8529
    environment:
      - ARANGO_NO_AUTH=1
    volumes:
      - arangodb-data:/var/lib/arangodb3
      - arangodb-apps:/var/lib/arangodb3-apps `

**Comment by jwierzbo** [comment]
@pritamlipu1 still nothing. 

Are you able to share with us your code? You mentioned before that you call `coll.EnsureTTLIndex()` - could you share what you do with this collection before this line? Could show what kind of data you keep there?

**Comment by paullesiak** [comment]
I was encountering the same issue while creating a persistent index on a collection with a very simple schema. I added a call to coll.IndexExists, and skip the index creation if it's already present, and that avoided this error... still not sure what the root cause is. FWIW, I was getting:

```
cannot unmarshal number 0.2 into Go value of type int
```

**Comment by paullesiak** [comment]
The issue seems to be due to selectivityEstimate being an integer instead of a float here:
https://github.com/arangodb/go-driver/blob/master/collection_indexes_impl.go#L46

@jwierzbo please let me know if I should open a PR

**Comment by jwierzbo** [comment]
Fixed in 1.3.2
